caiman/base/movies.py

```diff
@@ -1982,7 +2026,7 @@ def from_zip_file_to_movie(zipfile_name: str, start_end: Tuple = None) -> Any:
                         mov[counter] = np.array(Image.open(file))
 
                     if counter % 100 == 0:
-                        print([counter, idx])
+                        logging.debug([counter, idx])
```

Comment:
Let's clean this up a bit (not your fault, but since we're touching it); passing raw arrays to printing function means they're going to show up raw and ugly.